mods/gen4/random-teams.js

```diff
@@ -594,14 +596,18 @@ class RandomGen4Teams extends RandomGen5Teams {
 			item = 'Light Clay';
 		} else if (template.species === 'Palkia' && !!counter['Dragon'] && !!counter['Water']) {
 			item = 'Lustrous Orb';
+		} else if (ability === 'Serene Grace' && (hasMove['bodyslam'] || hasMove['headbutt'] || hasMove['ironhead'])) {
+			item = 'Leftovers';
```

Comment:
Is this necessary or can we just include Serene Grace in the higher Leftovers selection?

Reply:
Good call, I'll move this up.